app/bundles/CoreBundle/Resources/views/Helper/page_actions.html.twig

```diff
@@ -27,7 +27,7 @@
 
     {{ buttonsRender(
         (
-            '<button type="button" class="btn btn-default btn-nospin dropdown-toggle" data-toggle="dropdown" aria-expanded="false"><i class="ri-arrow-down-s-line"></i></button>\n'
+            '<button type="button" class="btn btn-default btn-nospin dropdown-toggle" data-toggle="dropdown" aria-expanded="false"><i class="ri-more-2-line"></i></button>\n'
```

Comment:
The page actions need to be reverted to the arrow down icon

```suggestion
            '<button type="button" class="btn btn-default btn-nospin dropdown-toggle" data-toggle="dropdown" aria-expanded="false"><i class="ri-arrow-down-s-line"></i></button>\n'
```

Reply:
Yes, that should be a down arrow.